tests/controllers/test_template_controller.py

```diff
@@ -136,12 +136,12 @@ def test_create_template(self, mock_db, mock_get_template_by_id, mock_get_classi
             template_controller.create_comms_template(template_id, template=template_object)
 
     @mock.patch('application.controllers.template_controller.db')
-    def test_get_template_by_classifier(self, mock_db):
+    def test_get_template_by_classifier_throws_exception(self, mock_db):
         # Given there is an error connecting with the database
         mock_db.session.query = mock.MagicMock(side_effect=SQLAlchemyError)
 
         # When we try to get a template by classifier
         # Then it raises a database error
         with self.assertRaises(DatabaseError):
-            template_controller.get_templates_by_classifiers(classifiers={"GEOGRAPHY": "NI",
-                                                                          "INDUSTRY": "construction"})
+            template_controller.get_comms_template_by_classifiers(classifiers={"GEOGRAPHY": "NI",
+                                                                               "INDUSTRY": "construction"})
```

Comment:
```JSON
{"GEOGRAPHY": "NI",
 "INDUSTRY": "construction",
})
```

Keeping a `,` at the end of dicts gives a cleaner look to future pull request code comparisons.